package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay
+??/??/?? IAlibay, JoseAntonioSiguenza
```

Comment:
You'll need to fix the merge conflicts on this and AUTHORS if possible.

Reply:
In general, thanks for the feedback, know I see the pep8speaks suggestions. Sorry for that.  I only have a question, could I resolve these conflicts online? I think it is because I'm working in "alingto" branch instead of "develop".